.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,24 @@
+name: BrowserStack Acceptance Tests
+
+on: [push, pull_request]
+
+jobs:
+  run-acceptance-tests:
+    name: BrowserStack Acceptance Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: BrowserStack Environment Setup
+        uses: 'browserstack/github-actions/setup-env@master'
+        with:
+          username:  ${{ secrets.BROWSERSTACK_USERNAME }}
+          access-key: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+          project-name: 'Answers Hitchhiker Theme'
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm ci
+      - run: npm run setup-test-site
+      - run: npm run build-test-site
+      - name: Run Acceptance Tests
+        run: 'npx testcafe "browserstack:chrome,browserstack:firefox,browserstack:safari,browserstack:ie@11" tests/acceptance/suites/*.js'
```

Comment:
Do we want to use headless browsers for chrome and firefox? I think with the current setup each PR into the theme will use all 8 of our threads. I think it's slightly better to use a browserstack browser, but would also feel bad about asking for another thread bump so soon.

Reply:
That works. I wasn't sure if our extra capacity would let us run all 4 browsers or not